src/tests/fixtures/projects/dummy-project.yaml

```diff
@@ -706,5 +706,5 @@ project-with-bigquery-remote-schemas:
                     remote_github:
                         schema: https://raw.githubusercontent.com/myrepo/something.json
                     local:
-                        schema: ./src/buildercore/bigquery/schemas/key-value.json
```

Comment:
I agree this should be shifted outside of `src/` to separate code and configuration; it's a fixture though so maybe the `fixtures` folder would be enough

Reply:
moved to ./src/tests/fixtures/